include/wx/xml/xml.h

```diff
@@ -222,6 +222,16 @@ enum wxXmlDocumentLoadFlag
     wxXMLDOC_KEEP_WHITESPACE_NODES = 1
 };
 
+// Create an instance of this and pass it to wxXmlDocument::Load()
+// to get detailed error information in case of failure. Values
+// are set to -1 if no error occurred.
+struct wxXmlParseError
+{
+    wxString message;
+    int line;
+    int column;
```

Comment:
I'd initialize those, i.e. do something like

```suggestion
    int line = 0;
    int column = 0;
```

BTW, you used -1 as "invalid" indicator, but as I think lines and columns are 1-based and not 0-based, IMO 0 makes more sense.